routes/api.php

```diff
@@ -30,4 +30,65 @@
     });
 });
 
-Route::get('/', 'UserController@all');
\ No newline at end of file
+Route::get('/', 'UserController@all');
+
+
+//// Products permissions
+Route::group(['middleware' => 'auth:api'], function(){
+   Route::group(['middleware' => 'admin'], function (){
```

Comment:
You can add multiple middleware to one route "Route::group(['middleware' => ['auth:api', 'businessActiveClient', 'orderAheadEnabled']]"